src/Microsoft.IdentityModel.Tokens/AsymmetricSignatureProvider.cs

```diff
@@ -124,6 +138,7 @@ public AsymmetricSignatureProvider(SecurityKey key, string algorithm, bool willC
 
             _minimumAsymmetricKeySizeInBitsForSigningMap = new Dictionary<string, int>(DefaultMinimumAsymmetricKeySizeInBitsForSigningMap);
             _minimumAsymmetricKeySizeInBitsForVerifyingMap = new Dictionary<string, int>(DefaultMinimumAsymmetricKeySizeInBitsForVerifyingMap);
+            _ecdsaKeySizeInBitsMap = new Dictionary<string, int>(DefaultECDsaKeySizeInBitsMap);
```

Comment:
Yes, you are right since I change DefaultECDsaKeySizeInBitsMap from 'public' to 'private'.